channelserver/src/main.rs

```diff
@@ -24,19 +24,19 @@ extern crate slog_async;
 extern crate uuid;
 #[macro_use]
 extern crate slog_term;
+extern crate maxminddb;
+extern crate uap_rust;
```

Comment:
It looks like this may be unused?

Reply:
Yes. odd that the dead code warning didn't flag it.